emcc.py

```diff
@@ -1679,6 +1679,14 @@ def include_and_export(name):
       # requires JS legalization
       shared.Settings.LEGALIZE_JS_FFI = 0
 
+    # For STANDALONE_WASM and MINIMAL_RUNTIME mode that memory is creating
+    # in the module and exported to JS.
+    # In trandiational emscripten mode we create it in JS and export it to
+    # wasm.  This allows us to set up memory, preload files, etc. even before
+    # the wasm module is created.
+    if shared.Settings.MINIMAL_RUNTIME or shared.Settings.STANDALONE_WASM:
+      shared.Settings.EXPORT_MEMORY_TO_WASM = 0
```

Comment:
The setting name is a bit confusing - `EXPORT_MEMORY_FROM_WASM` would more clearly explain what it does, if the value was flipped.

Reply:
Its an internal setting so its name is not that important.. its can't actually be specified on the command line, but I'll try it that way around and see how it the code flows.

Reply:
I went for `MEMORY_DEFINED_IN_WASM` .. that seems a little more clear. What do you think?

Reply:
Yeah I like that!